plasma_framework/contracts/src/exits/payment/controllers/PaymentStartInFlightExit.sol

```diff
@@ -352,8 +384,7 @@ library PaymentStartInFlightExit {
                 output.outputGuard,
                 exitData.outputGuardPreimagesForInputs[i]
             );
-            IOutputGuardHandler handler = exitData.controller.outputGuardHandlerRegistry.outputGuardHandlers(output.outputType);
-            assert(address(handler) != address(0));
+            IOutputGuardHandler handler = exitData.outputGuardHandlersForInputTxs[i];
```

Comment:
@kevsul  I would be very grateful if you could take a quick look on what's going on here.
I have a problem that happy path unit tests (contract as in this file) fail here. All checks in `verifyStart` are commented out, except `verifyOutputGuardHandlersForInputTxsRegistered`. 
I can't find a bug. Possibly it's the way array of output guard handlers it's being initialized or used.
I don't think it's worth spending much time on that, I can get rid off `IOutputGuardHandler` array.